server/static/static.go

```diff
@@ -120,10 +131,30 @@ func serveIndexTemplate(env environment.Env, tpl *template.Template, version str
 	}
 	err := tpl.ExecuteTemplate(w, indexTemplateFilename, &cfgpb.FrontendTemplateData{
 		Config:           &config,
-		JsEntryPointPath: *jsEntryPointPath,
+		JsEntryPointPath: jsPath,
 		GaEnabled:        !*disableGA,
 	})
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 	}
 }
+
+func AppBundleHash(bundleFS fs.FS) (string, error) {
+	hashBytes, err := fs.ReadFile(bundleFS, "sha.sum")
```

Comment:
could we call this something app-bundle-specific so it's not too confusing if we add some other kinda hash into the bundle later?

bundle.sha256 or app.sha256 or something?

Reply:
i'd like this to eventually be the sha of the full static bundle that we upload to gcs on each release, not just the js app bundle